migrations/1689065480_updateImageBlock.ts

```diff
@@ -0,0 +1,46 @@
+import { Client, SimpleSchemaTypes } from "@datocms/cli/lib/cma-client-node";
+
+export default async function (client: Client) {
+  const newFields: Record<string, SimpleSchemaTypes.Field> = {};
+
+  console.log("Creating new fields/fieldsets");
+
+  console.log(
+    'Create Single-line string field "Image position" (`image_position`) in block model "Image" (`image`)'
```

Comment:
I went for `image_position` because it's what we use on the [Section Image & Text block](https://voorhoede-website.admin.datocms.com/admin/blocks_library/2037669), but also thought that maybe `caption_position` might be clearer

Reply:
I think I would prefer `caption_position` indeed. I think in the context of the image & text block, it feels more natural to have image position, as IMO it's easier to understand the context in that the image position is relative to the text. But in this case I would say the image is the primary content. So it feels more natural to name it caption position, to stress that the position is relative to the image